### ISSUE TITLE
[Accessibility Improvement] Increase color contrast ratio

### ISSUE THREAD

**Issue by vavroom** [helpmeabstract/helpmeabstract#50]
The color contrast of several elements is particularly hard to see. Grey text on grey background may be trendy, but it's hard to read, particularly for people with low vision. But also for anyone trying to read the site on their mobile when they are outside.

Contrast ratio between foreground and background colors should be a minimum of 4.5:1. Best way to measure it is by using http://webaim.org/resources/contrastchecker/
